src/splitInteger.test.js

```diff
@@ -8,12 +8,23 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(splitInteger()).toEqual([]);
   });
 
   it(`should return an array with 'num' if parts = 1`, () => {
+    expect(splitInteger(8, 1)).toEqual([8]);
+  });
+
+  it(`should return an array of parts`, () => {
+    expect(splitInteger(32, 6)).toEqual([5, 5, 5, 5, 6, 6]);
+  });
+
+  it(`should return 0 if num = 0`, () => {
+    expect(splitInteger(0, 2)).toEqual([0, 0]);
+  });
 
+  it(`should return empty array if parts = 0`, () => {
+    expect(splitInteger(4, 0)).toEqual([]);
   });
```

Comment:
These tests are not needed, the function accepts only positive integers and all input should be valid